package.json

```diff
@@ -100,7 +92,20 @@
     "lint": "node ./node_modules/eslint/bin/eslint.js ./static/js",
     "scss_lint": "node ./node_modules/sass-lint/bin/sass-lint.js --verbose --no-exit",
     "test": "./js_test.sh",
-    "coverage": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- --compilers js:babel-register static/js/global_init.js 'static/**/*/*_test.js'",
+    "coverage": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --report lcovonly -- --compilers js:babel-register static/js/global_init.js 'static/**/*/*_test.js' && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
     "flow": "flow check"
+  },
+  "devDependencies": {
+    "eslint": "^2.13.1",
+    "eslint-config-defaults": "9.0.0",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-flow-vars": "^0.4.0",
+    "eslint-plugin-flowtype": "^2.3.0",
+    "eslint-plugin-react": "5.0.1",
+    "istanbul": "1.0.0-alpha.2",
+    "coveralls": "^2.11.14",
+    "mocha": "2.4.5",
+    "mocha-lcov-reporter": "^1.2.0",
+    "webpack-dev-server": "1.14.1"
```

Comment:
we don't use `devDependencies` because the version of packages in `devDependencies` cannot be locked down with `npm shrinkwrap`. please put things back as they were :)